docs/technical_documents/hybrid_a_star_en.md

```diff
@@ -0,0 +1,105 @@
+Generate coarse trajectory on the open space 
```

Comment:
make it a title? Please follow the template

Reply:
in the open space ? on the open space. Please stick to the former.